config.default.yaml

```diff
@@ -182,7 +182,9 @@ electricity:
   voltages: [132., 220., 300., 380., 500., 750.]
   co2limit: 7.75e+7 # European default, 0.05 * 3.1e9*0.5, needs to be adjusted for Africa
   co2base: 1.487e+9 # European default, adjustment to Africa necessary
-  agg_p_nom_limits: data/agg_p_nom_minmax.csv
+  agg_p_nom_limits:
+    file: data/agg_p_nom_minmax.csv
+    include_existing: false
```

Comment:
I wonder if it could make sense to add a comment explaining how the constraint will work for the sector-coupled model? My feeling is that it's perfectly fine to leave the current implementation focused on `Generator` component only [as wind and solar are likely to be the most demanded option in any case] we just need to make it evident for a user